src/api/create-object.js

```
@@ -23,28 +23,25 @@ const createObject = async ({
   const interceptorId = rax.attach();
   const ax = await axios({
     method: 'post',
-        url: isClient ? `${url}/${ApiVersion}/classes/${className}` : `${url}/${ApiVersion}/classes/${className}/userid/${userId}`,
+        url: isClient ? `${url}/${ApiVersion}/classes/${className}/objects` : `${url}/${ApiVersion}/classes/${className}/objects/userid/${userId}`,
         timeout: 10000,
         headers: {
           "Content-Type": "application/json",
           "Authorization": `Bearer ${token}`
         },
         data: object,
         raxConfig: {
-          httpMethodsToRetry: ['GET', 'PUT', 'HEAD', 'OPTIONS', 'DELETE', 'POST'] 
+          httpMethodsToRetry: ['GET', 'PUT', 'HEAD', 'OPTIONS', 'DELETE', 'POST']
         }
   })
   .then(response => response.data)
   .catch(error => {
-    console.log(error);
     if (error.response) {
```

Instead of logging, could we throw a descriptive error here? By catching and logging, we're effectively making it impossible to respond to failures. `throw` would at least allow consumers to react to the error.